internal/configtxgen/genesisconfig/config.go

```diff
@@ -254,10 +284,9 @@ func Load(profile string, configPaths ...string) *Profile {
 	}
 	logger.Debugf("Using config file: %s", config.ConfigFileUsed())
 
-	var uconf TopLevel
-	err = viperutil.EnhancedExactUnmarshal(config, &uconf)
+	uconf, err := loadConfigCache(config, config.ConfigFileUsed())
 	if err != nil {
-		logger.Panic("Error unmarshaling config into struct: ", err)
+		logger.Panic("Error loading config from config cache: %s", err)
```

Comment:
linter should yell about this - should be Panicf

Reply:
done